### ISSUE TITLE
[CheckStyle] Fix CheckStyle Warnings, Ensure we have 0 Warnings

### ISSUE THREAD

**Issue by Dee-Pac** [paypal/gimel#24]
Currently CheckStyle is set to Warning Level for several modules. This is because we have to do a lot of code cleanup to adhere to rules specified in checkstyle specification.

Without adding/changing any logic : only address the style sheet errors.

A plugin can be used to auto-fix the checkstyle violations. But this requires post-fix validation to ensure there is no inadvertent logic changes performed by any plugin that is used.

Other way is to fix manually each module.

Ensure the current code base adheres to the checkstyle by running the autofixer on the entire code base.

**Comment by Rishi55** [comment]
I would like to take this up!!

**Comment by Dee-Pac** [comment]
@Rishi55  Yes, This is still an open issue. Please let us know if you would like to work on this. We can assign the issue accordingly. Thanks
